### PR TITLE
Serialize jsonld

### PR DESCRIPTION
**Pull request by bourgeoa**

### CODE REVIEW

**Comment by bourgeoa** [comment]
@timbl @angelo-v 
The issue I have is that my patch returns a `Promise<string> and not a `String` 
To serialize `application/ld+json` in tests I need to make the the test `async` 'await serialize(...)

Have you any idea :
- how to resolve the Promise inside /rdflib/serialize.ts code
- if not is this kind of use acceptable  :  
```
async function serializeStore (store, contentType) {
const result = contentType === application/ld+json ? 
   await serialize(null, store, null, contentType) : 
   serialize(null, store, null, contentType)
}
```

File: tests/serialize/data.js [hunk 1]
```diff
@@ -122,7 +121,7 @@ var doNext = function (remaining) {
           })
           return
 
-          case '-dump':
+          /* case '-dump':
```
Comment (angelo-v) [inline tests/serialize/data.js:124]: let's remove unused code

File: src/serialize.ts [hunk 2]
```diff
@@ -114,4 +110,11 @@ export default function serialize (
       return result as string
     }
   }
+
+  function toJsonld (item ) {
+    try {
+      return jsonld.fromRDF(item, {format: 'application/n-quads'}).then( docJsonld => { return JSON.stringify(docJsonld) })
+      // return JSON.stringify(await jsonld.fromRDF(item, {format: 'application/n-quads'}))
```
Comment (angelo-v) [inline src/serialize.ts:117]: let's remove unused code

File: tests/unit/parse-test.js [hunk 3]
```diff
@@ -231,7 +235,70 @@ describe('Parse', () => {
       })
     })
 
-    describe('with a @type node', () => {
+    describe('jsonld with blanknodes', () => {
```
Comment (angelo-v) [inline tests/unit/parse-test.js:238]: Thanks for adding tests :+1:

**Review by angelo-v — APPROVED** [review]

**Pull request merged**